packages/cron/src/lib/utils.js

```diff
@@ -102,13 +123,34 @@ function getPgConnString (env, mode = 'rw') {
  * @param {Record<string, string|undefined>} env
  */
 export function getCargoPgPool (env) {
-  const connection = env.CARGO_PG_CONNECTION
-  if (!connection) {
+  let connectionString = env.CARGO_PG_CONNECTION
+  // if a connection string isn't specified try to build the connection string from other variables.
```

Comment:
I've left this, just to allow for easier setup locally? Less variables to set :P
But please disagree, and we can remove this.

Reply:
let's use the same vars in dev and prod!